sdk/compute/Azure.Management.Compute/README.md

```diff
@@ -1,69 +1,51 @@
-# README.md template
```

Comment:
I'm only editing Compute's README for now. Once we make sure its readme looks good, i'll do the same for other readmes